backend/danswer/background/celery/celery_utils.py

```diff
@@ -69,6 +69,30 @@ def get_deletion_attempt_snapshot(
     )
 
 
+def skip_cc_pair_pruning_by_task(
```

Comment:
better to name stuff like this something like "should_skip_cc_pair_pruning_by_task" to make the code a little more readable